charts/my-bloody-jenkins/README.md

```diff
@@ -137,6 +137,8 @@ The following table lists the configurable parameters of the chart and their def
 | `ingress.annotations`        | Ingress Annoations| `{}`
 | `ingress.hostname`        | Ingress Hostname (Required only if ingress is enabled)|
 | `ingress.tls.secretName`        | Ingress TLS Secret Name - if provided, the ingress will terminate TLS|
```

Comment:
I believe the documentation should state that secretName is mutual exclusive to certificate and privateKey.